packages/access-decision-manager-express/examples/blog-post--role-based-rbac/src/app.ts

```diff
@@ -0,0 +1,63 @@
+import AccessDecisionManagerProvider, { isGrantedMiddleware } from '@wizeline/access-decision-manager-express';
+import voters from './voters'
+import ATTRIBUTES from './attributes';
+
+import express = require('express');
+
+// Create a new express application instance
+const app: express.Application = express();
+
+const user = {
+  userId: 1,
+  attributes: {
+    role: 'editor'
+  }
+};
+
+app.use(AccessDecisionManagerProvider((req) => user, voters({})));
+
+app.get('/',
+  isGrantedMiddleware(ATTRIBUTES.PUBLIC),
```

Comment:
I think we want to change this to `ATTRIBUTES.GET_ALL_POST`, because that is the action that is being performed.

Remember we want to separate defining what the action is from who is allowed to do it.

Given this, I think we want to remove all references to `ATTRIBUTES.PUBLIC`.